jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/CurationSortingView.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from 'react'
```

Comment:
The curation sorting view.
![image](https://user-images.githubusercontent.com/3679296/101779038-bd5d3200-3ac2-11eb-9c53-7db6331d4a89.png)